fdbmonitor/fdbmonitor.cpp

```diff
@@ -428,21 +428,29 @@ struct Command {
 		const char* rd =
 		    get_value_multi(ini, "restart-delay", ssection.c_str(), section.c_str(), "general", "fdbmonitor", nullptr);
 		if (!rd) {
-			log_msg(SevError, "Unable to resolve restart delay for %s\n", ssection.c_str());
-			return;
-		} else {
-			max_restart_delay = strtoul(rd, &endptr, 10);
-			if (*endptr != '\0') {
-				log_msg(SevError, "Unable to parse restart delay for %s\n", ssection.c_str());
+			rd = get_value_multi(
```

Comment:
Since a lot of these calls go through `get_value_multi`, you may be able to extract the compatibility logic there.